### DISCUSSION TITLE
Automation team - access to all projects

### DISCUSSION THREAD

**Discussion by rkg-mm** [DependencyTrack/dependency-track#1425] [Q&A]
I am using the project access control. One issue I have is, when my automation user from build pipelines creates a new project version in Dtrack, it has no immediate permissions to access it. I manually need to give the Automation group permissions for this new project version.
However, the Administration group seems to have immediate access to all projects, without assigning projects. How is this achieved? 
Automation group already has permissions:
BOM_UPLOAD 
PORTFOLIO_MANAGEMENT 
PROJECT_CREATION_UPLOAD 
VIEW_PORTFOLIO 

With portfolio management, shouldn't it have access to the complete portfolio anyway?

**Comment by JorisVanEijden** [comment]
That would be ACCESS_MANAGEMENT
